### DISCUSSION TITLE
Out of all the docker I pulled on my synology why does this one the only one that works out of the box?

### DISCUSSION THREAD

**Discussion by dagarnx** [thijsvanloef/palworld-server-docker#99] [Q&A]
Out of all the docker I pulled on my synology why does this one the only one that works out of the box?

I'm curious if this container has something that makes it compatible with my synology as opposed to other containers?

**Comment by TangJames** [comment]
That's funny because when I tried to use this specific one on docker for synology, it crashes my whole DSM.

**Comment by dagarnx** [comment]
I’m in 1522+ with 7.2

Idk if my project compose would help but I can provide.

**Comment by dagarnx** [comment]
services:
   palworld:
      image: thijsvanloef/palworld-server-docker:latest
      restart: always
      container_name: palworld-server
      ports:
        - 8211:8211/udp
        - 27015:27015/udp
      environment:
         - ALWAYS_UPDATE_ON_START=true
         - PUID=1000
         - PGID=1000
         - PORT=8211 # Optional but recommended
         - PLAYERS=20 # Optional but recommended
         - MULTITHREADING=true
         - RCON_ENABLED=true
         - RCON_PORT=25575
         - ADMIN_PASSWORD="yourpassword"
         - COMMUNITY=true  
         - SERVER_PASSWORD="yourpassword"
         - SERVER_NAME="Your server name"
      volumes:
         - ./palworld:/palworld/
   rcon:
      image: outdead/rcon:latest
      entrypoint: ['/rcon', '-a', 'palworld:25575', '-p', 'adminPasswordHere']
      profiles: ['rcon']
